actions/image_partition_action.go

```diff
@@ -60,12 +60,8 @@ unique.
 
 'none' fs type should be used for partition without filesystem.
 
-- start -- offset from beginning of the disk there the partition starts.
-
-- end -- offset from beginning of the disk there the partition ends.
-
-For 'start' and 'end' properties offset can be written in human readable
-form -- '32MB', '1GB' or as disk percentage -- '100%'.
+- end -- offset from beginning of the disk there the partition ends or
+offset from beginning of where the partition starts specified with prefix "+"
```

Comment:
Rather then having end be either end point on the disk *or* the size of the disk it would make more sense to have a seperate size property and require either end or size to be provided.

size should be in similar to the units parted accepts (https://www.gnu.org/software/parted/manual/html_node/unit.html) ideally; The image size parsing is *mostly* there, only `%` and `s` really sould be handled. I don't think anyone still cares about cylinders :).

same parsing should probably be use for start and end rather then the current state were we just pass on whatever the user gives to parted and hope :).. 

That way it's both more clear for end-user, cleanup one of the debos oddities and more flexible then this implementation where there seems to be very specific  requirements between the start and end value "types"